packages/ember-toucan-core/src/components/form/fields/checkbox.ts

```diff
@@ -73,6 +73,10 @@ export interface ToucanFormCheckboxFieldComponentSignature {
      */
     value?: string;
   };
+  Blocks: {
+    label: unknown;
+    hint: unknown;
```

Comment:
`unknown` here - should these be `[]`?

Reply:
THIS was missing tests so I missed this! Should be fixed in the next round because I added tests.